vtexdocs/helpcenter
# Feat/faq

#### What is the purpose of this pull request?

To add the FAQ pages.

#### What problem is this solving?

- Added page that lists FAQs
- Added page to show FAQ docs
- Fixed position of the 'reading time' text in tracks and known issues.
- Fixed filter/sort button height/padding
- Fixed FAQ routes in sidebar, menu and hamburger menu

#### How should this be manually tested?

Open the preview and check if all the problems mentioned above are solved. Be sure to check the known issues and tracks pages, check all breakpoints and test the filter in the [FAQ listing page](https://deploy-preview-30--leafy-mooncake-7c2e5e.netlify.app/pt/faq).

#### Screenshots or example usage

![image](https://github.com/vtexdocs/helpcenter/assets/62757720/5412df04-308a-4d25-bf07-45ad05ee39c9)

![image](https://github.com/vtexdocs/helpcenter/assets/62757720/50b49c6c-c898-424f-87a1-dceb8e4f489e)

#### Types of changes
